defaults/main.yml

```diff
@@ -1,5 +1,11 @@
 ---
+postgres_version: 9.6
 postgres_locale: en_US.UTF-8
+postgres_extra_packages:
+  - python3-psycopg2
+  - python3-passlib
+  - pgtop
+  - autopostgresqlbackup
```

Comment:
I’m not sure this is a sane default for everyone that uses this. They may have other backup solutions.